app/uk/gov/hmrc/tai/util/yourTaxFreeAmount/EmploymentTaxCodeChangeReasons.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tai.util.yourTaxFreeAmount
+
+import com.google.inject.Inject
+import play.api.i18n.Messages
+import uk.gov.hmrc.tai.model.domain.TaxCodeChange
+import uk.gov.hmrc.tai.viewModels.taxCodeChange.{TaxCodePair, TaxCodePairs}
+
+class EmploymentTaxCodeChangeReasons @Inject()() {
```

Comment:
Like the use of lots of smaller functions. Does this need to be a class though. Would object suffice?